src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,20 @@
-export const TodoList = () => {};
+import React from 'react';
+import { TodoInfo } from '../TodoInfo';
+
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[]
+};
+
+// Add the required types and props
+export const TodoList: React.FC<Props> = ({ todos = [] }) => (
```

Comment:
You should not set a default value for `todos` prop. The `todos` prop is required in the Props